aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -84,6 +93,16 @@ module aptos_framework::aptos_governance {
         votes: Table<RecordKey, bool>
     }
 
+    /// Records partial voting proposals.
+    struct PartialVotingProposals has key {
```

Comment:
This looks only useful during the transitioning period but useless after once all proposals are "partial voting proposals". Is there a way to avoid needing this new struct?